bower.json

```diff
@@ -12,9 +12,7 @@
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.0.2",
     "ember-qunit": "0.1.8",
     "ember-qunit-notifications": "0.0.5",
-    "qunit": "~1.17.1"
-  },
-  "devDependencies": {
-    "highcharts-release": "~4.0.4"
+    "qunit": "~1.17.1",
+    "highstock-release": "~2.1.3"
```

Comment:
Did you mean to leave this in as a dependency? Or should users explicitly add the package(s) they want?